src/index.html

```diff
@@ -17,6 +17,12 @@
     />
```

Comment:
Stylesheets with a `.scss` extension won't work directly in HTML. You should compile your SCSS to CSS and then link to the CSS file instead.